src/dmd/target.h

```diff
@@ -37,6 +37,7 @@ struct Target
     // C ABI
     unsigned c_longsize;         // size of a C 'long' or 'unsigned long' type
     unsigned c_long_doublesize;  // size of a C 'long double'
+    uint criticalSectionSize;    // size of os critical section
```

Comment:
`unsigned`

Reply:
I should have simply made `unsigned` work in D.